dask/dataframe/core.py

```diff
@@ -5246,14 +5246,19 @@ def quantile(df, q, method="default"):
 
     # currently, only Series has quantile method
     if isinstance(df, Index):
-        meta = pd.Series(df._meta_nonempty).quantile(q)
+        series_typ = type(df._meta.to_series())
```

Comment:
```suggestion
        series_typ = df._meta.to_series()._constructor
```

I think cudf implements this now.